src/github/pull-request.ts

```diff
@@ -84,7 +83,7 @@ export const pullRequestWebhookHandler = async (context: CustomContext, jiraClie
 		return;
 	}
 
-	context.log(`Sending pull request update to Jira ${baseUrl}`);
+	context.log("Sending pull request update to Jira");
```

Comment:
Removed instead of hashed because it is already added on the `logger.child` call(line 28ish)